### DISCUSSION TITLE
Seeed Studio XIAO RP2040 with CircuitPython | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#375] [Q&A]
# Seeed Studio XIAO RP2040 with CircuitPython | Seeed Studio Wiki

Seeed Studio XIAO RP2040 with CircuitPython

https://wiki.seeedstudio.com/XIAO-RP2040-with-CircuitPython/

**Comment by Kasurni** [comment]
Can CircuitPython make colorful LEDs glow like MicroPython?
